motor/metaprogramming.py

```diff
@@ -149,6 +149,8 @@ def __init__(self, attr_name, doc=None):
 
     def create_attribute(self, cls, attr_name):
         name = self.attr_name or attr_name
+        if name == "encrypt_expression":
+            return
```

Comment:
You could instead add `None` as the last arg to `getattr()` and return if it is None.

Reply:
My concern with doing it that way is that we might forget to get rid of this workaround later. In general I think we do want to raise an error in this case (for example `Sync("method_that_is_mispelled"`), so I want it to be as specific as possible.

Reply:
This unconditionally gets rid of the attribute though right?  I think we need to first check if it exists, and then check if it is in an expected list of ones that may not exist yet.

Reply:
Ah, that is a very good point. Done.